documentation/bolt_running_tasks.md

```diff
@@ -3,72 +3,104 @@
 Bolt can run Puppet tasks on remote targets without requiring any Puppet
 infrastructure. 
 
-To execute a task, run `bolt task run`, specifying:
+To execute a task, specify the following:
 
 -   The full name of the task, formatted as `<MODULE::TASK>`, or as `<MODULE>`
     for a module's main task (the `init` task).
 -   Any task parameters, as `parameter=value`.
 -   The targets on which to run the task and the connection protocol, with the
-    `--targets` flag.
+    `targets` command-line option.
 -   If credentials are required to connect to the target, the username and
```

Comment:
```suggestion
-   (If credentials are required to connect to the target.) The username and
```
This is a nit, but this is the only sentence in the list that doesn't follow the same pattern